arrow/locales.py

```diff
@@ -1212,14 +1294,14 @@ class DutchLocale(Locale):
     day_abbreviations = ["", "ma", "di", "wo", "do", "vr", "za", "zo"]
 
 
-class SlavicBaseLocale(Locale):
-    def _format_timeframe(self, timeframe, delta):
+class SlavicBaseLocale(Locale, metaclass=ABCMeta):
+    timeframes: ClassVar[Mapping[TimeFrames, Union[str, Sequence[str]]]]
 
+    def _format_timeframe(self, timeframe: TimeFrames, delta: Union[float, int]) -> str:
         form = self.timeframes[timeframe]
         delta = abs(delta)
 
-        if isinstance(form, list):
-
+        if isinstance(form, Sequence):
```

Comment:
Can I modify like this? It feels more robust to me.

Reply:
I don't think so. It seems like `Sequence` encompasses more than `list` according to this: https://docs.python.org/3/glossary.html

Quote from the glossary: "Some built-in sequence types are list, str, tuple, and bytes"

Reply:
Got it! reverted it.